### PR TITLE
fix: updating links to TechDocs (Zoomin)

### PR DESCRIPTION
**Pull request by peknis**
Updating NCS links to point to the new TechDocs site.

### CODE REVIEW

File: docs/aws/GettingStarted/FirmwareConfiguration.rst [hunk 1]
```diff
@@ -26,4 +26,4 @@ After completing the configuration, compile the firmware either :ref:`using your
 
 .. note::
 
-   See the documentation on `nRF9160: Asset Tracker v2 application <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/asset_tracker_v2/README.html>`_ for all available configuration options.
\ No newline at end of file
+   See the documentation on `the Asset Tracker v2 application <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/asset_tracker_v2/README.html>`_ for all available configuration options.
```
Comment (wiba-nordic) [inline docs/aws/GettingStarted/FirmwareConfiguration.rst]:
```suggestion
   See the documentation on the `Asset Tracker v2 application <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/asset_tracker_v2/README.html>`_ for all available configuration options.
```

File: docs/azure/GettingStarted/FirmwareConfiguration.rst [hunk 2]
```diff
@@ -26,5 +26,5 @@ After completing the configuration, compile the firmware either :ref:`using your
 
 .. note::
 
-   See the documentation on `nRF9160: Asset Tracker v2 application <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/asset_tracker_v2/README.html>`_ for all available configuration options.
+   See the documentation on `Asset Tracker v2 application <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/asset_tracker_v2/README.html>`_ for all available configuration options.
```
Comment (wiba-nordic) [inline docs/azure/GettingStarted/FirmwareConfiguration.rst]:
```suggestion
   See the documentation on the `Asset Tracker v2 application <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/asset_tracker_v2/README.html>`_ for all available configuration options.
```

File: docs/guides/AutomateHEXFileBuilding.rst [hunk 3]
```diff
@@ -3,7 +3,7 @@
 Automate building of HEX files for your nRF Connect SDK application
 ###################################################################
 
-Continuous delivery shortens the time to market of a product. The nRF9160 DK supports firmware over-the-air (FOTA) upgrades `for AWS <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/aws_fota.html#lib-aws-fota>`_ and `for Azure <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/cellular/azure_fota/README.html>`_.
+Continuous delivery shortens the time to market of a product. The nRF9160 DK supports firmware over-the-air (FOTA) upgrades `for AWS <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/networking/aws_fota.html>`_ and `for Azure <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/networking/azure_fota.html>`_.
```
Comment (wiba-nordic) [inline docs/guides/AutomateHEXFileBuilding.rst]:
```suggestion
Continuous delivery shortens the time to market of a product.
The nRF9160 DK supports firmware over-the-air (FOTA) upgrades `for AWS <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/networking/aws_fota.html>`_ and `for Azure <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/networking/azure_fota.html>`_.
```

**Review by wiba-nordic — APPROVED** [review]
I have some tiny nit-picks, but I don't mind these changes going in as-is.

**Comment by coderbyheart** [comment]
What about the banner? https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/blob/ff33af1a543282a4963b23ba897e06c6a08cd8cd/conf.py#L55

**Comment by peknis** [comment]
> What about the banner?
Situation unchanged, still in beta.

**Review by coderbyheart — CHANGES_REQUESTED** [review]
Commit message does not follow project conventions, please change to 

`fix: updating links to TechDocs (Zoomin)`

**Review by coderbyheart — APPROVED** [review]

**Pull request merged**